src/tools/wasm-split.cpp

```diff
@@ -443,12 +446,50 @@ void instrumentModule(Module& wasm, const WasmSplitOptions& options) {
   writer.write(wasm, options.output);
 }
 
+std::set<Name> readProfile(Module& wasm, const WasmSplitOptions& options) {
```

Comment:
where is the profile file documented? I'm unsure what to compare the code here against.

Reply:
For now the only thing to compare against is the code that writes the profile, which is also in this file.

Reply:
Maybe it'd be good to add comments on the file format, however simple it is.